minseo/models/likesDao.js

```diff
@@ -0,0 +1,22 @@
+const appDataSource = require('./appDataSource');
+
+const createlikes = async (userId, postId) => {
+  try {
+    return await appDataSource.query(
+      `INSERT INTO likes(
+          user_id,
+          post_id
+          )VALUES(?,?);
+        `,
+      [userId, postId]
+    );
+  } catch (err) {
+    const error = new Error('INVALID_DATA_INPUT_LIKES');
+    error.statusCode = 500;
```

Comment:
상태코드 500은 예외처리가 되지 않았다는 것을 의미합니다. 따라서 예외처리를 진행한 코드에 대해서는 400번대 상태코드를 사용해 주세요.

```suggestion
    error.statusCode = 400;
```